homeassistant/components/lektrico/strings.json

```diff
@@ -0,0 +1,85 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Set required parameters to connect to your device",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "friendly_name": "[%key:common::config_flow::data::name%]"
+        }
+      },
+      "zeroconf_confirm": {
+        "description": "Do you want to add the Lektrico Charger with serial number `{serial_number}` to Home Assistant?",
+        "title": "Discovered Lektrico Charger device"
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
+    }
+  },
+  "entity": {
+    "sensor": {
+      "state": {
+        "name": "State"
+      },
+      "charging_time": {
+        "name": "Charging time"
+      },
+      "lifetime_energy": {
+        "name": "Lifetime energy"
+      },
+      "installation_current": {
+        "name": "Installation current"
+      },
+      "limit_reason": {
+        "name": "Limit reason"
+      },
+      "breaker_current": {
+        "name": "Breaker current"
+      },
+      "voltage_l1": {
+        "name": "Voltage L1"
+      },
+      "voltage_l2": {
+        "name": "Voltage L2"
+      },
+      "voltage_l3": {
+        "name": "Voltage L3"
+      },
+      "current_l1": {
+        "name": "Current L1"
+      },
+      "current_l2": {
+        "name": "Current L2"
+      },
+      "current_l3": {
+        "name": "Current L3"
+      },
+      "pf": {
+        "name": "PF"
+      },
```

Comment:
I think we can remove this translation key and fallback to the device class translation for Power factor

Reply:
I removed it, thanks.